.github/workflows/main.yaml

```diff
@@ -441,11 +437,22 @@ jobs:
         - name: Docker load kubeedge/build-tools image
           run: |
             docker load < /home/runner/build-tools/build-tools.tar
+            rm -rf /home/runner/build-tools/build-tools.tar
 
         - name: Enable cri config in containerd service
           run: |
             containerd config default | sudo tee /etc/containerd/config.toml && sudo systemctl restart containerd.service
 
+        - name: Increase disk space
+          uses: fcbg-platforms/increase-disk-space@latest
+          with:
+            tool-cache: true
+            android: true
+            dotnet: true
+            haskell: true
+            large-packages: true
+            old-dotnet: true
```

Comment:
Any docs for what's meaning of these fields?

Reply:
On the official page https://github.com/jlumbroso/free-disk-space, there isn't a detailed explanation, and it only states "Most of the options are self-explanatory."
![image](https://github.com/user-attachments/assets/963b2546-089b-4690-a143-95c41d501c3a)


However, it seems that:
- android, dotnet, haskell refer to whether you want to clean up libraries related to Android, .NET, and Haskell, respectively.
- docker-images indicates whether you want to clean up Docker images.
- swap-storage refers to whether to disable the swap partition.
- The large-packages option is not very clear regarding which contents it cleans up, but it doesn't seem to negatively impact the conformance workflow.

Reply:
Thanks, we can add `docker-images` in future if the disk space is still not enough.